templates/prometheus.go

```diff
@@ -19,11 +19,18 @@ package templates
 import (
 	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	"github.com/red-hat-storage/ocs-osd-deployer/utils"
+	corev1 "k8s.io/api/core/v1"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	ServingCertificateKeyVolumeName = "serving-cert"
+	ServiceCACertificateVolumeName  = "ca-cert"
+	KubeRbacProxyConfigVolumeName   = "kube-rbac-config"
```

Comment:
Please make these private. 
We should not export if we do not have to.

```suggestion
	servingCertificateKeyVolumeName = "serving-cert"
	serviceCACertificateVolumeName  = "ca-cert"
	kubeRbacProxyConfigVolumeName   = "kube-rbac-config"
```

Reply:
Not using variable anymore for these values and directly adding in the template